tests/testthat/test-age_helpers.R

```diff
@@ -0,0 +1,391 @@
+test_that("age_labels() works", {
+
+  # Test simple use case
+  expect_identical(
+    age_labels(c(5, 12, 20, 30)),
+    c("00-04", "05-11", "12-19", "20-29", "30+")
+  )
+
+  # Test superfluous zeros are ignored
+  expect_identical(
+    age_labels(c(0, 5, 12, 20, 30)),
+    c("00-04", "05-11", "12-19", "20-29", "30+")
+  )
+
+  # Test zero-padding works for large ages
+  expect_identical(
+    age_labels(c(5, 12, 20, 30, 130)),
+    c("000-004", "005-011", "012-019", "020-029", "030-129", "130+")
+  )
+})
+
+
+# Create a test data set that contains all birth dates in 2000, 1970, and 1940 (years with, or almost with, leap years)
+test_data <- c(
+  seq.Date(from = as.Date("2000-01-01"), to = as.Date("2000-12-31"), by = "1 day"),
+  seq.Date(from = as.Date("1970-01-01"), to = as.Date("1970-12-31"), by = "1 day"),
+  seq.Date(from = as.Date("1940-01-01"), to = as.Date("1940-12-31"), by = "1 day")
+) |>
+  tibble::tibble(birth_date = _) |>
+  dplyr::mutate(reference_date = birth_date + lubridate::days(40 * dplyr::row_number()))
+
+
+test_that("age_on_date() works for date input", {
+
+  test_date <- as.Date("2024-02-28")
+
+  for (conn in get_test_conns()) {
+
+    # Copy to the remote
+    test_ages <- dplyr::copy_to(conn, test_data, "test_age")
+    SCDB::defer_db_cleanup(test_ages)
+
+    # Compute the reference age using lubridate
+    reference_ages <- test_data |>
+      dplyr::mutate(age = floor(lubridate::interval(.data$birth_date, test_date) / lubridate::years(1)))
```

Comment:
Is this exact?
How about using 

lubridate::as.period(lubridate::interval(<same>), unit = "years)
?

Reply:
I just checked a different syntax which gives the same results (the tests still succeed at least)

```
lubridate::year(lubridate::as.period(lubridate::interval(.data$birth_date, test_date)))
```

Maybe this is more readable? I am not sure what I prefer the most.

Reply:
The latter is more readable IMO. 
(To me it is not clear how 'interval(x,y)/years(1)' handles leap days and the like which I find is more clear when avoiding the division)